reactor/ip_addr_changed.sls

```diff
@@ -0,0 +1,26 @@
+{%- set api_servers = salt['pillar.get']('roles:kube-master', '') %}
+{%- set sender      = data['id'] %}
+
+{% if sender in api_servers %}
+
+# one of the API servers has changed IP:
+# updpate the "<ip> api api.infra.caasp.local" in the minions
+update_hosts_minions:
+  local.state.apply:
+    - tgt: 'G@roles:kube-minion'
+    - expr_form: compound
```

Comment:
I will try both things once we have #44 merged (with `hosts-master` and `hosts-minion`)...